Bento/Bento/Section.swift

```diff
@@ -35,7 +35,7 @@ public struct Section<SectionID: Hashable, ItemID: Hashable> {
                             .footer: AnyRenderable(footer)]
     }
 
-    internal init(id: SectionID, items: [Item], supplements: [Supplement: AnyRenderable]) {
+    public init(id: SectionID, items: [Item], supplements: [Supplement: AnyRenderable]) {
```

Comment:
This will allow to put `supplements` for `UITableView` which can have only header and footer

Reply:
Right, but we already have `public func adding<R: Renderable>(_ supplement: Supplement, _ component: R) -> Section`

Reply:
True